musi/pk.srt

```diff
@@ -2595,207 +2595,271 @@ please.
-where is he?
+ona li lon seme?
+==where is he?
 
 616
 01:12:51,699 --> 01:12:53,309
-Save me!
+o weka e mi tan ni!
+==Save me!
 
 617
 01:13:02,255 --> 01:13:07,151
-Somebody help!
+jan o pana e pona!
+==Somebody help!
 
 618
 01:13:52,096 --> 01:13:58,755
-I was in the himalaya..meditating
+mi lon nena Imalaja... li pilin e sewi.
+==I was in the himalaya..meditating
 
 619
 01:13:59,191 --> 01:14:11,298
-..then suddenly i see light,
+ni la suno li kama
+tan ijo lili ni lon anpa.
+==..then suddenly i see light,
 from this small object on the ground.
 
 620
 01:14:11,493 --> 01:14:19,236
-I closed my eyes thought about God,
+mi pini e oko mi li pilin e sewi.
+sewi li suno e mi.
+==I closed my eyes thought about God,
 and i got enlighted.
 
 621
 01:14:19,509 --> 01:14:27,952
-this object was so hot even in
+ijo ni li seli mute. taso ona
+li lon lete mute.
+==this object was so hot even in
 a very cold weather.
 
 622
 01:14:28,080 --> 01:14:35,795
-God confirmed its the broken piece of
+sewi li toki e ni: ijo ni li wan tan
+ilo kalama pi lawa Siwa. o kama jo e ona.
+==God confirmed its the broken piece of
 lord shiva's pellet drum. embrace it.
 
 623
 01:14:36,017 --> 01:14:48,306
-take it and put it in a temple. its auspicious
+li pana e ona lon tomo sewi. lukin ona li kama
+e pona tawa jan li weka e ike tan jan.
+==take it and put it in a temple. its auspicious
 sight will take people's trouble away.
 
 624
 01:14:49,985 --> 01:14:57,459
-would you like to see it?
+sina wile ala wile lukin e ona?
+==would you like to see it?
 
 625
 01:15:00,108 --> 01:15:04,242
-praise the lord shiva.
+o sewi e lawa Siwa.
+==praise the lord shiva.
 
 626
 01:15:17,759 --> 01:15:23,156
-Lord shiva, miracle!
+lawa Siwa o! kama sewi!
+==Lord shiva, miracle!
 
 627
 01:15:23,156 --> 01:15:24,267
-stop.
+o pini!
+==stop.
 
 628
 01:15:24,267 --> 01:15:28,571
-Let him come, he's excited.
+o pini ala e ona. ona li pilin wawa.
+==Let him come, he's excited.
 
 629
 01:15:28,711 --> 01:15:31,489
-Lord Shiva, Greetings
+lawa Siwa o, pona!
+==Lord Shiva, Greetings
 
 630
 01:15:31,489 --> 01:15:33,711
-not there, here
+o tawa ni ala. o kama.
+==not there, here
 
 631
 01:15:33,949 --> 01:15:36,383
-Say what you want to say here.
+o kama li toki e ijo pi wile sina.
+==Say what you want to say here.
 
 632
 01:15:36,383 --> 01:15:42,630
-Shiva is great.
+sewi Siwa li suli.
+==Shiva is great.
 -Shiva is great.
 
 633
 01:15:42,759 --> 01:15:46,967
-Lord where are you hiding?
+lawa o, sina lon seme?
+o kama lon ni li kama suli tan toki mi.
+==Lord where are you hiding?
 come upfront, accept my gratitude.
 
 634
 01:15:47,759 --> 01:15:54,342
-He's right here.
+ona li lon ni!
+mi lukin e ona lon tomo pi ko jaki.
+mi wile kama jo e ijo mi la ona li tawa wawa.
+==He's right here.
 found him in the toilet.
 when i asked for my property, he ran.
 
 635
 01:15:54,584 --> 01:16:03,182
-I thought he escaped but he brought me here
+mi la ona li weka. taso ona li pana e mi lon ni

```

Comment:
"mi la" is *"lon mi la", but "tawa mi la" is explained in pu, whereas "mi la" isn't.
→ tawa mi la